src/hammer-vlsi/defaults.yml

```diff
@@ -312,6 +312,25 @@ vlsi.inputs:
     #  - y (int) - The Y coordinate of the bump assigned to this net
     assignments: []
 
+  # Pin placement mode. (str)
+  # Specifies how to arrange pins for your design.
+  # Valid options:
+  # none - This mode lets the CAD tool do whatever it wants (providing no constraints).
+  #   Typically this is sane but unpredictable.
+  # generated - Use the assignments list below
+  # auto - is unsupported but should look at hierarchy and guess where pins should go
+  pin_mode: none
+  pin:
+    # List of PinAssignment Structs.
+    #   You must specify pins and one of either macro or layers and side.
+    # - pins (str) - The name(s) of pins that will be assigned.
```

Comment:
Maybe call this `name`? Not sure how this works if this is multiple pins

Reply:
It is intentionally plural because you can specify multiple pins at once see the test. I can call it names if you would like

Reply:
Yeah that sounds good. Maybe name it `names` and say multiple pins can be specified via wildcards.

Reply:
Just for clarity for future users, a short comment that one can use `*` for wildcarding the pins would be nice.

Reply:
Done.